src/main/java/com/adamk33n3r/runelite/watchdog/SoundPlayer.java

```diff
@@ -54,51 +84,71 @@ public void processQueue() {
     }
 
     public void play(File soundFile, int volume) {
-        this.queue.add(Pair.of(soundFile, volume));
+        this.play(soundFile, volume, 0);
+    }
+
+    public void play(File soundFile, int volume, int repeatTime) {
+        this.queue.add(new SoundItem(soundFile, volume, repeatTime));
         if (!WatchdogPlugin.getInstance().getConfig().putSoundsIntoQueue()) {
-            SwingUtilities.invokeLater(() -> this.playNext(this.createMP3Player()));
+            SwingUtilities.invokeLater(() -> this.playNext(this.mp3Player));
```

Comment:
I just realized that with the change of not creating a new mp3 player for every sound, this prevents multiple mp3s from being played at the same time. It will interrupt the currently played sound to play the new one. Definitely a regression.

Reply:
This code is so gnarly, not yours, mine. Lol. I need to rewrite it...

Reply:
Yeah I noticed that too. We'd need to dynamically create more MP3Players each time a new sound is started. Which creates a whole different mess (and just feels wrong).

Reply:
It used to do that, with the `createMP3Player`